code/datums/hud/human.dm

```diff
@@ -993,17 +993,14 @@
 			health_oxy.icon_state = "moxy[stage]"
 			health_oxy.tooltipTheme = "healthDam healthDam[stage]"
 
-			// may as well let you see you're being irradiated if you can already see individual things like oxy/tox/burn/brute
-			//update_rad_indicator(master.radiation ? 1 : 0)
```

Comment:
won't this prevent you from seeing rad indicators if you have a health implant?

Reply:
Nope 
![image](https://user-images.githubusercontent.com/3855802/229350787-311944a6-5064-435d-a20a-d0fbfe98c204.png)

Reply:
but I think I should remove those calls anyway, I don't think they're supposed to be there